### DISCUSSION TITLE
Should have a 'community' link in navbar

### DISCUSSION THREAD

**Discussion by sawyerbivens** [community/community#27363] [Other Feature Feedback, Questions, & Ideas]
Personally I find the community page to be rather annoying to get to without just typing the URL into the search bar. I feel like it would be helpful towards getting more community involvement and generally making the community discussions more accessible.

Perhaps like here:
![image](https://user-images.githubusercontent.com/48366954/181356707-f9a013a2-8d62-48da-bd5a-eb4181866a90.png)

**Comment by jfullstackdev** [comment]
bookmark the URL and it's just one click away
